### PR TITLE
Enhancing Consistency Chapter with Demo Videos

### PR DESCRIPTION
**Pull request by paulowoicho**
### Summary

Fixes #<NUM>

This PR adds demo videos to the chapter on maintaining consistency, add another medium for contributors to learn about the consistency checks.

### List of changes proposed in this PR (pull-request)

- [x] Added demo videos for checks that can be demonstrated

### What should a reviewer concentrate their feedback on?

- [x] The initial plan was to use a gif, however, gif making tools (such as GIPHY) have a 40s time constraint on video length. Suggestions for creating longer gifs are most welcome! We can then replace these videos with the new gifs in another PR
- [x] I have tried to make the videos responsive, please check if it works on smaller viewports (other than a desktop screen)
- [x] The videos are hosted on my Youtube account. If possible, they need to be moved to The Turing Way's youtube channel instead.
- [x] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by malvikasharan** [comment]
I love the video addition. So exciting! Would you like to create a playlist on turning way YouTube channel to post these videos?

**Comment by paulowoicho** [comment]
> I love the video addition. So exciting! Would you like to create a playlist on The Turing Way YouTube channel to post these videos?

@malvikasharan, yes please! I was going to request access to The Turing Way's youtube channel so I could move the videos there

**Comment by malvikasharan** [comment]
I have uploaded all the videos in this playlist: https://www.youtube.com/watch?v=HxcdqKJbCE4&list=PLBxcQEfGu3DnzbPz652KN9fRnn3CPzgkw

**Comment by malvikasharan** [comment]
There were some conflict errors which I have fixed, but please double-check if everything looks ok before merging.

File: book/website/community-handbook/consistency/consistency-formatting.md [hunk 1]
```diff
@@ -102,6 +108,11 @@ Thus, we recommend that you format your Checklist subchapters as unordered lists
 - Item Three
 
 ```
+#### Demo
+
+<div class="video-container">
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/ujsasdW1boI" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (malvikasharan) [inline book/website/community-handbook/consistency/consistency-formatting.md:114]:
```suggestion
<div class="video-container">
    <iframe width="560" height="315" src="https://www.youtube.com/embed/oe2Up1pU5DY" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

File: book/website/community-handbook/consistency/consistency-formatting.md [hunk 2]
```diff
@@ -124,6 +135,12 @@ Ideally, all Markdown files should start with a level 1 heading and increase seq
 Several files in _The Turing Way_ book do not follow this convention. 
 A list of such files can be found in [this issue](https://github.com/alan-turing-institute/the-turing-way/issues/1321), and [this PR](https://github.com/alan-turing-institute/the-turing-way/pull/1451) is a great example of how to fix a file with non-consecutive headers.
 
+#### Demo
+
+<div class="video-container">
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/t-RR-y6fQ5U" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (malvikasharan) [inline book/website/community-handbook/consistency/consistency-formatting.md:141]:
```suggestion
    <iframe width="560" height="315" src="https://www.youtube.com/embed/qq9QCrykdbw" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

File: book/website/community-handbook/consistency/consistency-formatting.md [hunk 3]
```diff
@@ -134,6 +151,13 @@ This helps make _The Turing Way_ more navigable and accessible.
 
 {ref}`Cross-referencing <ch-style-crossref>` is discussed in detail in the {ref}`ch-style-guide`. The subchapter explains what labels are, provides a naming convention for labels in _The Turing Way_, and gives several useful examples for how cross-referencing should be done.
 
+#### Demo
+
+<div class="video-container">
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/ZnZdpnzwDZ8" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (malvikasharan) [inline book/website/community-handbook/consistency/consistency-formatting.md:157]:
```suggestion
    <iframe width="560" height="315" src="https://www.youtube.com/embed/ikcjxjklLVg" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

File: book/website/community-handbook/consistency/consistency-formatting.md [hunk 4]
```diff
@@ -159,6 +183,14 @@ Please refer to the {ref}`style guide <ch-style-figures>` for examples on format
 When including images in your contributions, it may be better to avoid the height parameter as the wrong value could make your image appear distorted on mobile devices.
 You should always check how your image looks in the Netlify preview of the book when you make a PR.
 
+#### Demo
+
+<div class="video-container">
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/P0NFyZ8AaOg" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (malvikasharan) [inline book/website/community-handbook/consistency/consistency-formatting.md:189]:
```suggestion
    <iframe width="560" height="315" src="https://www.youtube.com/embed/upBiKLR_A5E" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

File: book/website/community-handbook/consistency/consistency-formatting.md [hunk 5]
```diff
@@ -190,6 +222,12 @@ The `_toc.yml` is the file where the book-wide table of contents for _The Turing
 
 Nonetheless, remember that the final title should adequately tell readers what to expect from a chapter or subchapter.
 
+#### Demo
+
+<div class="video-container">
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/K6lDmd9X6yw" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (malvikasharan) [inline book/website/community-handbook/consistency/consistency-formatting.md:228]:
```suggestion
    <iframe width="560" height="315" src="https://www.youtube.com/embed/HxcdqKJbCE4" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

File: book/website/community-handbook/consistency/consistency-formatting.md [hunk 6]
```diff
@@ -211,3 +249,11 @@ For example, In {ref}`<mismatched_title_toc>` above, **Using spreadsheets for re
 
 Certain headers may not need to be title-cased depending on the context in which they are used.
 For example, because some of the headers in this chapter make up a checklist - they do not need to be title-cased.
+
+#### Demo
+
+<div class="video-container">
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/gKrno6Dev-Q" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (malvikasharan) [inline book/website/community-handbook/consistency/consistency-formatting.md:256]:
```suggestion
    <iframe width="560" height="315" src="https://www.youtube.com/embed/ET_LI5dwP9M" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

File: book/website/community-handbook/consistency/consistency-language.md [hunk 7]
```diff
@@ -24,6 +24,12 @@ This could help minimize the number of grammatical errors that make it to the fi
 With respect to tone, ensure that chapters adhere to a formal style of writing and that sentences are easy to digest.
 One rule of thumb to consider is that if a sentence needs to be read more than once to be understood, then it likely needs to be rephrased.
 
+#### Demo
+
+<div class="video-container">
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/x6cbUq2vALE" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (malvikasharan) [inline book/website/community-handbook/consistency/consistency-language.md:30]:
```suggestion
    <iframe width="560" height="315" src="https://www.youtube.com/embed/Prv23kGekVY" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

File: book/website/community-handbook/consistency/consistency-structure.md [hunk 8]
```diff
@@ -76,6 +76,15 @@ alt: A subchapter in The Turing Way with three table of contents. Two that are g
 ---
 The manually written table of contents is unnecessary as Jupyter Book auto-generates two separate table of contents.
 ```
+#### Demo
+
+<div class="video-content">
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/okJYzwzzHuI" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (malvikasharan) [inline book/website/community-handbook/consistency/consistency-structure.md:82]:
```suggestion
    <iframe width="560" height="315" src="https://www.youtube.com/embed/zKWrvgCxSB0" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

File: book/website/community-handbook/consistency/consistency-formatting.md [hunk 9]
```diff
@@ -70,6 +70,12 @@ For example, if you want to add a Youtube video to your content using the `<ifra
 
 ````
 
+#### Demo
+
+<div class="video-container">
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/vLIGy1OVrUc" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (malvikasharan) [inline book/website/community-handbook/consistency/consistency-formatting.md:76]:
```suggestion
    <iframe width="560" height="315" src="https://www.youtube.com/embed/tv0HlVgxDdI" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

**Review by malvikasharan — APPROVED** [review]
Updated the youtube link.

**Review by paulowoicho — COMMENTED** [review]
I really like the thumbnails on the videos, They're awesome! Everything looks great as well so I'll go ahead and merge

**Pull request merged**